ClangSharp/Generated/CXLanguageKind.cs

```diff
@@ -3,8 +3,8 @@ namespace ClangSharp
     public enum CXLanguageKind
     {
         CXLanguage_Invalid = 0,
```

Comment:
An example of the native header explicitly setting the value for one entry, but not the subsequent declarations